tests/hipify-clang/lit.site.cfg.in

```diff
@@ -7,8 +7,10 @@ config.cuda_root = "@CUDA_TOOLKIT_ROOT_DIR@"
 if sys.platform in ['win32']:
     config.cuda_sdk_root = "@CUDA_SDK_ROOT_DIR@"
     if not config.cuda_sdk_root or config.cuda_sdk_root == "CUDA_SDK_ROOT_DIR-NOTFOUND":
-        config.cuda_samples_root = os.environ.get('NVCUDASAMPLES_ROOT')
-        if not config.cuda_samples_root or config.cuda_samples_root == "NVCUDASAMPLES_ROOT-NOTFOUND":
+        cuda_version = "@CUDA_VERSION@"
+        cuda_version = cuda_version.replace('.','_')
+        config.cuda_samples_root = os.environ.get('NVCUDASAMPLES' + cuda_version + '_ROOT')
+        if not config.cuda_samples_root:
```

Comment:
should we check here that `config.cuda_samples_root` is defined and is not empty, but has value ended with "-NOTFOUND"?

Reply:
No, we shouldn't cause NVCUDASAPLESX_Y_ROOT is not inquired by cmake.

Reply:
NVCUDASAMPLES_ROOT is not inquired as well, thus the removed check was also superfluous.